dcrjson/dcrwalletextresults.go

```diff
@@ -128,6 +128,26 @@ type StakePoolUserInfoResult struct {
 	InvalidTickets []string         `json:"invalid"`
 }
 
+// StartAutoBuyerResponse models the data returned from the startautobuyer
+// command.
+type StartAutoBuyerResponse struct {
```

Comment:
There is no reason to have an empty type for a response and also even if it were this is not named appropriately since everything else is named `...Result`.

Really, the entire second commit can be removed as it's all not needed.